contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -4659,6 +4671,19 @@ static bool set_and_persist_temp_oid_buffer_start(Oid new_oid)
 	return true;
 }
 
+static bool
+pltsql_is_local_only_inval_msg(const SharedInvalidationMessage *msg)
+{
+	return (msg->id == SHAREDINVALRELCACHE_ID && msg->rc.local_only);
+}
+
+static bool
+pltsql_is_enr_locktag(const LOCKTAG *locktag)
```

Comment:
instead of this introduce a hook for get_ENR_withoid, locktag type can be checked on engine itself after hook null check